docs/preview/continuous-patching/README.md

```diff
@@ -0,0 +1,477 @@
+Continuous Patching Workflow in Azure Container Registry
+========================================================
+
+## Introduction
+
+Azure Container Registry’s Continuous Patching feature automates the detection and remediation of operating system(OS) level vulnerabilities in container images. By scheduling regular scans with [Trivy](https://trivy.dev/) and applying security fixes using [Copa](https://project-copacetic.github.io/copacetic/website/), you can maintain secure, up-to-date images in your registry—without requiring access to source code or build pipelines. Simply customize the schedule and target images to keep your Azure Container Registry(ACR) environment safe and compliant
+
+> [!NOTE] 
+> Continuous Patching is in Private Preview as of October 2024.
+
+
+## Use Cases
+
+Here are a few scenarios to use Continuous Patching:
+
+- **Enforcing container security and hygiene:** Continuous Patching enables users to quickly fix OS container CVEs without the need to fully rebuild from upstream.
+- **Speed of Use:** Continuous Patching removes the dependency on upstream updates for specific images by updating packages automatically. Vulnerabilities can appear every day, while popular image publishers may only offer a new release once a month. With Continuous Patching, you can ensure that container images within your registry are patched as soon as the newest set of OS vulnerabilities are detected.
+
+## Preview Limitations
+
+Continuous Patching is currently in private preview. The following limitations apply:
+- Windows-based container images aren’t supported.
+- Only "OS-level" vulnerabilities that originate from system packages will be patched. This includes system packages in the container image managed by an OS package manager such as “apt” and “yum”. Vulnerabilities that originate from application packages, such as packages used by programming languages like Go, Python, and NodeJS are unable to be patched.  
+- End of Service Life (EOSL) images are not supported by Continuous Patching. EOSL images refer to images where the software provider is no longer offering updates, security patches, and technical support. Examples include Debian 8 and Fedora 28. EOSL images will be skipped from the patch despite having vulnerabilities - the recommended approach is to upgrade your image to a supported version.
```

Comment:
End of Service Life (EOSL) images are not supported by Continuous Patching. EOSL images refer to images where the underlying operating system is no longer offering updates, security patches, and technical support. Examples include images based on older operating system versions such as Debian 8 and Fedora 28. EOSL images will be skipped from the patch despite having vulnerabilities - the recommended approach is to upgrade your the underlying operating system of your image to a supported version.